auth/oidc.go

```diff
@@ -60,17 +60,35 @@ var (
 	// Request parameter to specify the OpenID Connect provider to be used for authentication,
 	// from the list of providers defined in the Sync Gateway configuration.
 	OIDCAuthProvider = "provider"
+
+	// MaxProviderConfigSyncInterval is used as the duration between now and next provider
```

Comment:
It looks like most of these (as well as OIDCAuthProvider) are intended to be constants, not variables.  The only reason OIDCDiscoveryRetryWait is defined as a var is that tests need the ability to modify it - that doesn't seem to be the case for the rest of these.

Reply:
Makes sense, marked all these variables are constants except OIDCDiscoveryRetryWait.